src/usr/local/www/system_advanced_admin.php

```diff
@@ -597,4 +597,4 @@
 	log_error(gettext("webConfigurator configuration has changed. Restarting webConfigurator."));
 	send_event("service restart webgui");
 }
-?>
\ No newline at end of file
+?>
```

Comment:
Alright. This shouldn't cause any trouble either. Because that file is not included anywhere, but rather a directly accessed page.